extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosInterface.java

```diff
@@ -179,6 +178,10 @@ String uploadPhotoContent(InputStream inputStream, @Nullable String sha1)
           .encodeToString(BaseEncoding.base16().decode(sha1.toUpperCase())));
     }
 
+    if(!title.isEmpty())
```

Comment:
nit: style here is a bit weird - I'm guessing this was just wonkiness from your new IDE setup